Source/Core/Core/HW/VideoInterface.cpp

```diff
@@ -52,7 +52,9 @@ static UVIBorderBlankRegister    m_BorderHBlank;
 // 0xcc002080 - 0xcc002100 even more unknown
 
 u32 TargetRefreshRate = 0;
-
+int VIWidth = 640;
+int VIHeight = 480;
+bool VMode = false;
```

Comment:
Passing variables via globals is not a great idea, along with giving them defaults which only make sense for ntsc games.

This code is a bit of a mess, really you should pass the aspect ratio into Video_BeginField, but if virtual/real XFB is disabled, the frame can be sent to the screen long before Video_BeginField is actually called.

A compromise would be to add a VideoInterface::GetAspectRatio method to VideoInterface and the video backend can call it to get the current aspect ratio. When XFB is disabled, the video backend can live with the consequences of asking for the aspect ratio early.

XFB disabled is a total hack anyway, maybe it should just have incorrect aspect ratios.